src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -415,8 +415,9 @@ def fuzz(self,
            extra_env=None):
     """LibFuzzerCommon.fuzz override."""
     self._test_qemu_ssh()
-    self.fuzzer.start([])
-    self.fuzzer.monitor()
+
+    ret = self.fuzzer.start([])
```

Comment:
nit: rename ret to something more meaningful.

also, can we pass libfuzzer arguments (i.e. additional_args) here?

Reply:
renamed to return_code

i added a todo for the libfuzzer arguments, if you don't mind?  it's just that the testing of this has been delicate / finnicky enough that i'm really nervous about adding more stuff that may-or-may-not cause issues, and i'd rather isolate those in a different PR